shape-blog-react/static/app/utils.js

```diff
@@ -0,0 +1,9 @@
+import _kebabCase from 'lodash/kebabCase';
+import qs from 'query-string';
+
+export const getImageUrl = (path, params ={}) => `${'https://takeshape-dev.imgix.net'}/${path}?${qs.stringify(params)}`;
```

Comment:
These three functions will be replaced with equivalents from `takeshape-routing` which should be published this morning.